### DISCUSSION TITLE
Customizing resource form fields location. How to?

### DISCUSSION THREAD

**Discussion by cleargoal** [filamentphp/filament#12942] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

Hi everybody!

I would like to have the form as shown in the screenshot. 
![image](https://github.com/filamentphp/filament/assets/10929782/fd1f86b4-f22d-4cdc-9be1-be9cfacb8852)

The field I struck in red was added by me to wrap date fields on the same row. Actually, I don't need that field and want to have 
In the 1st row - 3 fields
In 2nd and 3d - 2.

Is it possible?

Any assistance is very appreciated.

**Comment by dissto** [comment, marked as answer]
You could use a group component. There is also many other ways though.

```php
return $form
  ->schema([
      \Filament\Forms\Components\Group::make([
          TextInput::make('sdfsdf'),
          TextInput::make('sdfsdf'),
          TextInput::make('sdfsdf'),
      ])
          ->columns(3)
          ->columnSpanFull(),

      \Filament\Forms\Components\Group::make([
          TextInput::make('sdfsdf')
              ->columnSpan(1),
          TextInput::make('sdfsdf')
              ->columnSpan(1),
      ])
          ->columns(3)
          ->columnSpanFull(),

      \Filament\Forms\Components\Group::make([
          TextInput::make('sdfsdf')
              ->columnSpan(1),
          TextInput::make('sdfsdf')
              ->columnSpan(1),
      ])
          ->columns(3)
          ->columnSpanFull(),
  ]);
  
  ```
  
  Would give you: 
![Screenshot 2024-05-26 110943](https://github.com/filamentphp/filament/assets/11778632/84580c0b-1430-4453-87da-4bcd5a321a23)

Or 
```php
return $form
  ->schema([
  
      \Filament\Forms\Components\Group::make([
          TextInput::make('sdfsdf'),
          TextInput::make('sdfsdf'),
          TextInput::make('sdfsdf'),
      ])
          ->columns(3)
          ->columnSpanFull(),
  
      \Filament\Forms\Components\Group::make([
          TextInput::make('sdfsdf'),
          TextInput::make('sdfsdf'),
      ])
          ->columns(3)
          ->columnSpanFull(),
  
      \Filament\Forms\Components\Group::make([
          TextInput::make('sdfsdf'),
          TextInput::make('sdfsdf'),
      ])
          ->columns(3)
          ->columnSpanFull(),
  ]);
  
  ```
Would give you: 
![Screenshot 2024-05-26 110917](https://github.com/filamentphp/filament/assets/11778632/e87c35d7-3d6e-46ef-a225-09600039cc90)

**Reply by cleargoal** [reply]
Great!
Thank you very much!
It's exactly what I need.
